cmd/addimport.go

```diff
@@ -444,6 +447,13 @@ func (p *AddImportParams) PostInteractive(ctx ActionCtx) error {
 		return err
 	}
 
+	if !p.service {
+		p.allowTrace, err = cli.Confirm("allow tracing", false)
```

Comment:
For service export, you did that in "PreInteractive" and here you are doing for "PostInteractive", any reason for that?

Reply:
Yes, in the case where they are creating the import from an activation token or by picking from some export, some of the options may or not exist. By PostInteractive() the question can be asked for that.

Reply:
ok